content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -304,7 +304,17 @@ mvn -Dset.changelist \
 
 == Troubleshooting
 
-Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets and that your pull request has a label indicating it was of interest to users (e.g., enhancement rather than chore).
+Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets.
+
+=== No release has been triggered after merging a pull request
+
+Ensure that the pull request has a label indicating it was of interest to users (e.g., `enhancement` rather than `chore`).
+
+See the complete list of label categories at https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml and the list of "interesting" categories at https://github.com/jenkins-infra/interesting-category-action/blob/main/action.yaml#L13.
```

Comment:
I think it would be good to explicitly mention `developer` and explain that it should be used for a PR which does nothing on its own but is a dep of a downstream PR currently using an incremental version, so that the downstream can switch to a release version and be taken out of draft.

Reply:
Should be addressed in https://github.com/jenkins-infra/jenkins.io/pull/7686/commits/41d2f2099afd0ea1b11ea7e4dd5c448738a739b8